File: DAL/Models/Team.cs
using System;
using System.Collections.Generic;

namespace DAL.Models
{
    public class Team : BaseEntity
    {
        public string TeamName { get; set; }
        public string Discription { get; set; }
        public ICollection<TeamAndTeamMembers> TeamMemmbers { get; set; }
        public Guid? CompanyId { get; set; }
        public virtual Company Company { get; set; }
        public bool? Status { get; set; }     
    }
}

File: WFM/ClientApp/src/app/views/models/Team.ts
import { TeamMemberForTeamPage } from "./TeamMemberForTeamPage"
import { TeamMember } from "./teamMember"

export class Team {
    id: string
    teamName: string
    discription: string
    Status: boolean
    createdAt: string
    teamMemmbers: TeamMemberForTeamPage[];
    userId: string;
    companyId: string;
}